### DISCUSSION TITLE
Fixing `ghub-repository-id: Repository "org/repo" does not exist on "api.github.com".

### DISCUSSION THREAD

**Discussion by anegg0** [magit/forge#678] [Support] [answered]
Hi there,
After some time failing to add my organization's repos with `forge-add-repository` (returned:  `ghub-repository-id: Repository "org/repo" does not exist on "api.github.com".`). I fixed the issue by creating a fine-grained GitHub token and clearing the `authinfo` cache.
I thought this problem might affect other users, so I wanted to mention my use-case.
Thank you for magit and forge, @tarsius.

**Comment by tarsius** [comment, marked as answer]
What made the difference here isn't that you created a *fine-grained* token -- the previous token was also fine-grained, as other tokens are not supported. More likely the old token had simply expired.

**Comment by anegg0** [comment]
Noted, thanks.

-------- Original Message --------
